### ISSUE TITLE
Missing Accessibility Features - Friendly Feedback

### ISSUE THREAD

**Issue by TTomczek** [TTomczek/studygroupmap#21]
Hi there! I love using the studygroupmap project, it's super helpful. I just wanted to kindly point out that I noticed some accessibility features seem to be missing. It would be great to have those improvements to make the app even more inclusive for everyone. Thanks for considering this! Keep up the great work!
